src/modules/telemetry/main.ts

```diff
@@ -15,10 +13,10 @@ export default (
         LSSM.$t(`modules.telemetry.${key}`, args);
 
     const checkBrowser = (browser: UAParser.IBrowser, browserMajor: number) =>
-        LSSM.$store
-            .dispatch('storage/get', {
+        LSSM.$stores.storage
+            .get<string>({
                 key: HIDE_BROWSER_NOTE_KEY,
-                defaultValue: false,
+                defaultValue: '',
```

Comment:
On purpose?

Reply:
yes, because it is stored as a string instead of a boolean